x-pack/heartbeat/cmd/root.go

```diff
@@ -24,7 +24,7 @@ var RootCmd *cmd.BeatsRootCmd
 // configuration generated from a raw Elastic Agent config
 func heartbeatCfg(rawIn *proto.UnitExpectedConfig, agentInfo *client.AgentInfo) ([]*reload.ConfigWithMeta, error) {
 	//grab and properly format the input streams
-	inputStreams, err := management.CreateInputsFromStreams(rawIn, "metrics", agentInfo)
+	inputStreams, err := management.CreateInputsFromStreams(rawIn, "synthetics", agentInfo)
```

Comment:
The only outstanding thing for this PR is to make a similar change for packetbeat and auditbeat, which both default to "logs" instead of "metrics"

https://github.com/elastic/beats/pull/33921#discussion_r1038283527